source/less/blocks/additional-products.less

```
@@ -0,0 +1,32 @@
+.additional-products {
+  background-color: #ffffff;
+  padding: 25px 0 35px 0;
+}
+
+.additional-products__title {
+  font-family: "oswaldregular", sans-serif;
+  font-size: 24px;
+  line-height: 24px;
+  font-weight: 400;
+  color: #000000;
+  margin-top: 0;
+  margin-bottom: 36px;
+}
+
+.additional-products__advertising-text {
+  background-image: url("../../img/background-catalog-advertise.png");
```

```suggestion
  background-image: url("../img/background-catalog-advertise.png");
```
Иначе сервер не найдёт изображение.